client/my-sites/stats/stats-list/stats-list-item.jsx

```diff
@@ -196,17 +197,12 @@ module.exports = React.createClass( {
 				icon = ( <span className="stats-list__flag-icon" style={ style } /> );
 			}
 
-			return (
-				<span className={ wrapperClassSet } key={ i } >
-					{ gridiconSpan }
-					{ icon }
-					<Emojify>
-						<a onClick={ this.preventDefaultOnClick } href={ this.props.data.link } >
-							{ labelItem.label }
-						</a>
-					</Emojify>
-				</span>
-				);
+			itemLabel = ( <Emojify>{ labelItem.label }</Emojify> );
+			if ( this.props.data.link ) {
+				itemLabel = ( <a onclick={ this.preventDefaultOnClick } href={ this.props.data.link } >{ itemLabel }</a> );
```

Comment:
I think it's the right decision to not apply `Emojoify` to the text node here as the links are going to be encoded & largely occluded by the surrounding UI anyhow. 👍